src/components/skipNavLink.js

```diff
@@ -0,0 +1,42 @@
+import { div, h } from 'react-hyperscript-helpers'
+import { Link } from 'src/components/common'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+
+
+export const SkipNavLink = Utils.forwardRefWithName('SkipNavLink', (props, ref) => {
+  return h(Link, {
+    as: 'a',
+    href: '#',
+    className: 'reveal-on-focus',
+    style: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      background: 'white',
+      color: colors.accent(),
+      border: '1px solid black',
+      borderColor: colors.accent(),
```

Comment:
Just curious why you did this instead of:
```suggestion
      border: '1px solid',
      borderColor: colors.accent(),
```

Reply:
Habit, I think. ;-)  Thanks for pointing it out